setup.py

```diff
@@ -47,7 +47,8 @@
     'base58',
     'googlefinance',
     'requests_futures',
-    'PyYAML'
+    'PyYAML',
+    'envparse'
```

Comment:
You've probably got to add this to the [windows list](https://github.com/lbryio/lbry/blob/838508f342d96bac0e3a3b7458c12b7bbac10963/setup.py#L244-L244) as well